bundles/ch.elexis.core.data/src/ch/elexis/data/Anwender.java

```diff
@@ -198,18 +196,19 @@ protected Anwender(final String id){
 	 * Initializes the first user to the system, the administrator
 	 */
 	protected static void initializeAdministratorUser(){
-		new User();
-		
 		Anwender admin = new Anwender();
 		admin.create(null);
 		admin.set(new String[] {
 			Person.NAME, FLD_LABEL, Kontakt.FLD_IS_USER
 		}, ADMINISTRATOR, ADMINISTRATOR, StringConstants.ONE);
-		User.load(ADMINISTRATOR).setAssignedContact(admin);
-		
-		CoreHub.actUser = admin;
-		ElexisEventDispatcher.getInstance().fire(
-			new ElexisEvent(admin, Anwender.class, ElexisEvent.EVENT_USER_CHANGED));
+		IUser user = CoreModelServiceHolder.get().load(ADMINISTRATOR, IUser.class).orElse(null);
+		if (user != null) {
+			user.setAssignedContact(
```

Comment:
Eigentlich darf er hier gar nie null sein oder doch??

Reply:
wird beim start ausgeführt.  und könnte null sein wenn in der User_ Tabelle kein admin existiert. lass uns das mal bereden.

Reply:
kein null check benötigt.